internal/storage/blob/cloner.go

```diff
@@ -20,68 +19,57 @@ import (
 	"github.com/cerbos/cerbos/internal/util"
 )
 
-// clonerFS represents file system interface that used by the Cloner.
 type clonerFS interface {
 	fs.FS
 	Remove(name string) error
 	Create(name string) (io.WriteCloser, error)
 	MkdirAll(path string, perm fs.FileMode) error
 }
 
-type infoType map[string][]byte
+type (
+	etagType      string
+	fileNameType  string
```

Comment:
These types are not quite necessary because we are not passing these values around to other functions.